projects/fusion-e2e/tests/pages/alert-page.ts

```diff
@@ -0,0 +1,53 @@
+import {AlertComponent} from '../components/alert/alert-component';
+import {alertStoryId, defaultTestId, loadedPageSelector} from '../components/alert/consts';
+import {Page} from '@playwright/test';
+import {BasePage} from './base-page/base-page';
+import {ComponentProps} from './base-page/types';
+
+export class AlertPage extends BasePage {
+    readonly component: AlertComponent;
+
+    constructor(page: Page) {
+        const alertProps: ComponentProps = {
+            page: page,
+            testId: defaultTestId,
+            componentId: alertStoryId,
+            loadedPageSelector: loadedPageSelector
+        };
+
+        super(alertProps);
+        this.component = new AlertComponent(page, this.testId);
+    }
+
+    getAlertText() {
```

Comment:
No need to wrap the same functions if you inherit them from the component class

Reply:
it's on the page, so we need to use the component class

Reply:
buttonPage.button.click

change component property name to button